src/main/java/net/sf/jabref/model/database/DuplicationChecker.java

```diff
@@ -64,38 +72,41 @@ public int getNumberOfKeyOccurrences(String key) {
 
     }
 
-    //========================================================
-    // keep track of all the keys to warn if there are duplicates
-    //========================================================
-    public boolean addKeyToSet(String key) {
+    /**
+     * Keep track of all the keys to warn if there are duplicates
```

Comment:
The description is quite confusing when looking at the method name, please change it.